### PR TITLE
replace null-cap CSR reset values with a tag clear (fixes issue 43)

### PR DESCRIPTION
**Pull request by tariqkurd-repo**
All CSRs which used to reset to NULL_CAP now just have a tag clear with unspecified data.
All CSRs which reset to infinity are unchanged

### CODE REVIEW

**Review by arichardson — COMMENTED** [review]
This looks reasonable to me. @jrtc27 @sorear do you agree?

Commit message should include "Fixes https://github.com/riscv/riscv-cheri/issues/43"?

**Comment by sorear** [comment]
I think that we will eventually need an ARC position on whether "reset values" apply before or after the execution of an implementation-dependent BootROM and where the appropriate place for compatibility requirements is.

Within the assumptions that have been made previously on this tracker, LGTM.

**Comment by tariqkurd-repo** [comment]
https://github.com/riscv/riscv-cheri/pull/101 still has JVTC as infinity but fixes everything else to do with reset values.

File: src/riscv-integration.adoc [hunk 1]
```diff
@@ -545,7 +545,9 @@ include::img/mscratchreg.edn[]
 ==== Machine Scratch Register Capability (mscratchc)
 
 The <<mscratchc>> register is an extension to <<mscratch>> that is able to hold
-a capability. Its reset value is the <<null-cap>> capability.
+a capability.
+
+{TAG_RESET_CSR}
```
Comment (andresag01) [inline src/riscv-integration.adoc:550]: Could you please also fix the text of these other CSRs?
* mepcc sepcc: Currently states that the reset value is NULL
* sscratchc

**Review by andresag01 — CHANGES_REQUESTED** [review]

File: src/riscv-integration.adoc (hunk 1 above)
Comment (tariqkurd-repo) [reply src/riscv-integration.adoc:550]: yes ok - thanks for checking

**Pull request merged**